bedrock/products/urls.py

```diff
@@ -20,8 +19,14 @@
         # Pages that do not use allowed_countries or default_monthly_price contexts
         page('vpn/desktop', 'products/vpn/platforms/desktop.html',
              ftl_files=['products/vpn/platforms/shared', 'products/vpn/platforms/desktop']),
-    )
-else:
-    urlpatterns += (
-        redirect(r'^vpn/desktop(/.*)?', 'products.vpn.landing', permanent=False),
+        page('vpn/desktop/linux', 'products/vpn/platforms/linux.html',
+             ftl_files=['products/vpn/platforms/shared', 'products/vpn/platforms/linux']),
```

Comment:
We should probably reorder these files as the first one is used for activation (same applies below)
```suggestion
             ftl_files=['products/vpn/platforms/linux', 'products/vpn/platforms/shared']),
```

Reply:
Done!